packages/babel-runtime/package.json

```diff
@@ -6,12 +6,12 @@
   "repository": "https://github.com/babel/babel/tree/master/packages/babel-runtime",
   "author": "Sebastian McKenzie <sebmck@gmail.com>",
   "dependencies": {
-    "core-js": "^2.1.0"
+    "core-js": "^2.1.0",
+    "regenerator-runtime": "^0.9.5"
```

Comment:
Yeah I guess so, and we would manually update babel-runtime when regenerator changes?